shared/actions/config/index.tsx

```diff
@@ -223,15 +223,12 @@ function* loadDaemonAccounts(
       )
     }
 
-    // only reload in the user-switching case
-    const loadConfiguredAccountsAgain = state.config.loggedIn && flags.fastAccountSwitch
```

Comment:
whoops, I guess this is overly restrictive. I'll come up with a new heuristic, I guess, so as to not be calling this super duper often.